packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -519,10 +533,6 @@ class _InteractiveViewerState extends State<InteractiveViewer> with TickerProvid
   // https://github.com/flutter/flutter/issues/57698
   final bool _rotateEnabled = false;
 
-  // Used as the coefficient of friction in the inertial translation animation.
-  // This value was eyeballed to give a feel similar to Google Photos.
-  static const double _kDrag = 0.0000135;
```

Comment:
Can you actually keep this constant and the comments here?  And then use the constant as the default value above instead of hard coding it.